src/components/Link.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { routeTo } from 'redux-router-kit';
+
+const LinkView = ({ targetTo, text, path }) =>
+  (<a
+    tabIndex="0"
```

Comment:
We can make the `tabIndex` configurable with default value `0` by adding a property `tabIndex = 0` to `LinkView` arguments. Like so:

```
const LinkView = ({ targetTo, text, path, tabIndex = 0 }) =>
```